mcs/class/Mono.Messaging/Mono.Messaging_test.dll.sources

```diff
@@ -1,3 +1,5 @@
 Mono.Messaging/ConcurrentLinkedQueueTest.cs
 Mono.Messaging/QueueReferenceTest.cs
-
+../../../build/common/Consts.cs
+../../../build/common/Locale.cs
+AssemblyInfo.cs
```

Comment:
You need to add the new files like MessagingProviderLocatorTests here. The csproj is auto-generated and changes there won't be persisted.